brian_cheung/the_basics.md

```diff
@@ -114,14 +114,14 @@ the index value of Michelangelo would be 2 because an array starts from 0 not 1
 
 **Answer 8A:**
 ```ruby
-puts jurassic_park{"Triceratops"}
+puts jurassic_park["Triceratops"]
 ```
 
 * In Ruby, write code to add "Dr. Satler" to the "Humans" of `jurassic_park`.
 
 **Answer 8B:**
 ```ruby
-jurassic_park{"Humans"}.push"Dr. Satler"
+jurassic_park["Humans"].push"Dr. Satler"
 ```
 
```

Comment:
This is closer, but still won't work. Try running it in IRB or pry - what errors do you get?